scripts/ssl-enum-ciphers.nse

```diff
@@ -1149,6 +1151,15 @@ action = function(host, port)
     return nil
   end
 
+  -- Store the discovered ciphers in host table, should be utilized by other
+  -- scripts to find cipher suite available.
+  for key, val in pairs(results) do
+    host.registry.tls_protos[key] = {}
```

Comment:
We should be careful of 2 things: first, don't assume ssl-enum-ciphers is the only script that will ever add to this key, so do something like `host.registry.tls_protos = host.registry.tls_protos or {}` to avoid clobbering it. Second, different services may have different lists of supported protocols, so we should probably key this by service. Something like `host.registry.tls[port.number .. port.protocol][version] = {"TLS_RSA_BLAHBLAH"}`